fcl/detsim/detsim_2d_icarus.fcl

```diff
@@ -14,28 +15,38 @@ services: {
 physics: {
 
   producers: {
+    @table::icarus_shifting_triggersim.producers
+
     crtdaq:         @local::icarus_crtsim
      opdaq:         @local::icarus_simpmt
        daq:         @local::icarus_simwire_wirecell
 
        rns:         { module_type: "RandomNumberSaver" }
   } # producers
-  
-  simulate: [ rns, opdaq, daq, crtdaq ]
-  
+
+  simulate: [ rns, opdaq, @sequence::icarus_shifting_triggersim.path, daq, crtdaq ]
+
   # define the output stream, there could be more than one if using filters
   stream:  [ rootoutput ]
 
 } # physics
 
-
 outputs: {
-  rootoutput: @local::icarus_rootoutput
+  rootoutput: {
+    @table::icarus_rootoutput
+    outputCommands: [
+      "keep *",
+      "drop *_ionization_*_*",
+      "drop *_pmtlvdsgatesinit_*_*",
+      "drop *_pmtfixedthrinit_*_*",
+      "drop *_pmttriggerwindowsinit_*_*",
+      "drop *_triggersimgatesinit_*_*"
+      ]
+  }
 }
 
 #legacy G4 configs
 services.Geometry.GDML: "icarus_complete_20220518_overburden.gdml"
 services.Geometry.ROOT: "icarus_complete_20220518_overburden.gdml"
-physics.producers.daq.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel.jsonnet"]
-physics.producers.crtdaq.G4ModuleLabel: "largeant"
-physics.producers.opdaq.InputModule: "largeant"
+physics.producers.daq.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-shifted.jsonnet"]
+physics.producers.crtdaq.G4ModuleLabel: "shifted"
```

Comment:
@PetrilloAtWork, I have resolved the merge conflicts, which I think makes it clearer what is actually being changed by this PR in these lines.